overlord/ifacestate/ifacemgr_test.go

```diff
@@ -377,8 +377,7 @@ slots:
 	s.mockIface(c, &interfaces.TestInterface{InterfaceName: "test"})
 
 	setup()
-
-	//mgr := s.manager(c)
+	_ = s.manager(c)
```

Comment:
@mvo5 it cannot be in a common setup as there is some code that has to run before a manager is constructed.